task/common/values.go

```diff
@@ -41,7 +41,15 @@ type Event struct {
 	Code        string
 	Description []string
 }
-type Storage struct{}
+type RemoteStorage struct {
+	// Container stores the id of the container to be used.
+	Container string
+	// Path stores the subdirectory inside the container.
+	Path string
```

Comment:
Can we merge those two so users can specify either `storage` or `storage/prefix` with a single string?

Reply:
I'm hesistant to do so - it feels like separating the two explicitly is clearer. Especially since we'll most likely also need to add a `container-type` field to support pre-allocated buckets in k8s.

Reply:
In Kubernetes, this field would probably take a user-specified StorageClass, which is a single string. 🤔 What else could it take?

Reply:
what about e.g. a minio bucket?

Reply:
That's not a native storage strictly speaking, and would have the same consideration as a Google Cloud task using a Microsoft Azure storage account. It would be out of scope for us, and users would be responsible of synchronization as part of their code.